features.md

```diff
@@ -16,19 +16,67 @@
    
 * ### Viewing Patients
 
-   1. Downloading Patients
```

Comment:
Why did you remove the above header?

Reply:
Changing this to a small header makes it clearer. [See here](https://github.com/openmrs/openmrs-android-client-user-guide/pull/17/files#diff-e1ec7fcfd5aa4d3ea3e990d9b707a3b4R21)